hyeonjun/13week/68645.py

```diff
@@ -0,0 +1,35 @@
+def solution(n):
+    answer = []
+    down_right = [[] for _ in range(n)]  # 아래진행 + 우측진행만 저장하는 배열
+    up = [[] for _ in range(n)]  # 위로진행만 저장하는 배열
+
+    ceiling = 0
+    floor = n-1
+    num = 1
+
+    while n:
```

Comment:
```suggestion
    while n > 0:
```
이렇게 넣으면 밑에 if문 종료조건을 다 없애도 될 것 같습니다.